guides/browser-support.md

```diff
@@ -0,0 +1,171 @@
+
+### Automation
+
+Thankfully the world has created tools to help us track and polyfill
+disparities between browsers. We rely on a handful of of these tools to help
+us provide a level of backward compatability for modern features. As mentioned
+above, this doesn't necessarily mean feature parity. Where it's impossible or
+impractical to implement a modern feature, we fallback to standard practices
+for that browser. For example, we do not deliver interactive scripting for
+Internet Explorer 8, but we do ensure that default browser features continue to
+work so users that can't or don't want to upgrade continue to have access to
+the site and our content.
+
+#### Configuration
+
+Each project has it's own configuration depending on the requirements of the
+tools and features that project depends on. As a general rule we strive to
+cover the following browser list in each project.
+
+We configure [Autoprefixer](#autoprefixer) and [Babel](#babel) to support
+the following list of browsers.
+
+- Latest 2 releases of all browsers including:
+    - Chrome
+    - Firefox
+    - Safari
+    - Internet Explorer
+    - Edge
+    - Opera
+    - iOS Safari
+    - Opera Mini
+    - Android Browser
+    - BlackBerry Browser
+    - Opera Mobile
+    - Chrome for Android
+    - Firefox for Android
+    - Samsung Internet
+- Internet Explorer 9
+
+http://browserl.ist/?q=last+2+versions%2C+Explorer+%3E%3D+9
+
+As well as additional Autoprefixer support for:
+
+- Internet Explorer 8
+
+http://browserl.ist/?q=last+2+versions%2C+Explorer+%3E%3D+8
+
+### Browser Testing
+
+We use automated tests with a headless version of Chrome to ensure the majority
+of the site works as expected. For manual testing, we realistically test this
+project locally or in a virtual environment with the following list of
+browsers:
+
+- Chrome
+- Firefox
+- Safari
+- Internet Explorer 8, 9, 10, and 11
+- Edge
+- iOS Safari
+- Chrome for Android
+
+#### Individual projects to test
+
+- cfgov-refresh
+- Capital Framework
+- Design Manual
+- Paying for College
+- Retirement
+- Complaint
+- Eregs
+- Salesforce/Mosaic/Submit a complaint app/Complaint portal
+- cfpb-chart-builder
+- HMDA Platform
+- Claiming social security
+
+### Definitions
+
+#### Autoprefixer
+
+Autoprefixer parses our CSS and adds vendor prefixes to rules where necessary
+using reported feature support by [Can I Use](https://caniuse.com/). For more
+information visit the [Autoprefixer documentation site]
+(https://autoprefixer.github.io/).
+
+#### Babel
+
+Babel compiles our [ES6](http://es6-features.org/) JavaScript where necessary
+for the browsers that either don't support or have limited support of ES6
+features. For more information visit the [Babel documentation site]
+(https://babeljs.io/).
+
+#### Interactive JavaScript
+
+Interactive scripting is additions or adjustments to the browsers' default
+response to a user's interaction. Examples are the Mega Menu, Expandables,
+Notifications, and Sortable Tables.
```

Comment:
Notifications, strictly speaking, do not necessarily require scripting.

Reply:
Neither does the Mega Menu or Expandables. If scripting is turned off they still fallback to a usable experience.

Reply:
I suppose that's true, but notifications are regularly used in non-Ajax-form-submission scenarios, whereas the mode of the others is _always_ enhanced by JS, if it's available.

Reply:
But the changes to Notifications being injected instead of a page load, that is an adjustment to the browser's default response. I think this is correct because that line exists prior to it and I'd rather be clear and lay out exactly what components we typically modify vs those we don't.

Reply:
That's not always the case, e.g., https://www.consumerfinance.gov/consumer-tools/financial-well-being/, where it's always just on the page, but this is a really minor point and I don't think it will be confusing to anyone.